cilium/cmd/helpers_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build privileged_tests
+// +build !privileged_tests
```

Comment:
Unfortunately I believe that this is privileged because the endpoint package directly depends on BPF bits, which depend on the bootstrapping of the BPF filesystem, which causes this test to fail in unprivileged mode on Travis. More invasive refactoring would be required to loosen this requirement.

Reply:
We can still have 2 files, one that runs privileged tests and other that doesn't run privileged tests.

Reply:
The problem here is that the imports need the bpf mount, even if the test would not use it. The dev VM has that mount so it worked for me locally, but it failed for this reason in Travis. I reverted this change.